openpype/hosts/resolve/startup.py

```diff
@@ -44,17 +46,22 @@ def open_file(path):
 def main():
     # Open last workfile
     workfile_path = os.environ.get("OPENPYPE_RESOLVE_OPEN_ON_LAUNCH")
-    if workfile_path:
+
+    if workfile_path and os.path.exists(workfile_path):
+        log.info(f"Opening last workfile: {workfile_path}")
         open_file(workfile_path)
     else:
-        print("No last workfile set to open. Skipping..")
+        log.info("No last workfile set to open. Skipping..")
```

Comment:
Do these logs even show anywhere? Last time I tried I didn't see this in the `console` output of Resolve.